product_packaging_unit_price_calculator/tests/test_product_price_packaging_qty.py

```diff
@@ -36,22 +37,19 @@ def setUpClass(cls):
         )
 
     def test_set_sale_package_price(self):
-        form = Form(self.wizard_1)
-        form.packaging_price = 200
-        form.selected_packaging_id = self.pkg_box
+        self.wizard_1.packaging_price = 200
+        self.wizard_1.selected_packaging_id = self.pkg_box
         self.assertEqual(self.wizard_1.current_unit_price, 222)
-        self.assertEqual(form.unit_price, 4)
-        form.save()
+        self.assertEqual(self.wizard_1.unit_price, 4)
         self.wizard_1.action_set_price()
         self.assertEqual(self.product.list_price, 4)
 
     def test_set_purchase_pacakge_price(self):
         self.wizard_1.product_supplierinfo_id = self.supplier_info
-        form = Form(self.wizard_1)
+        self.wizard_1.product_tmpl_id = self.product.product_tmpl_id
```

Comment:
@hailangvn Do you have tested with the 'with' context manager form? As the onchanges won't apply with the previous code?

Reply:
Thanks @rousseldenis for the question. Here are my findings.
* onchanges will be applied when we access the column in `depends` statement so the previous code does work;
* the original test passed with the `with` context manager and without it;
* this change has no positive effect so I am closing this.